packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/Filtering/FilterFlyout.js

```diff
@@ -292,6 +311,7 @@ const FilterFlyout = ({
               components.RadioButtonGroup.onChange?.(selected);
             }}
           >
+            {<RadioButton id="any" labelText="Any" value="Any" />}
```

Comment:
no need for the curly braces
```suggestion
            <RadioButton id="any" labelText="Any" value="Any" />
```